### PR TITLE
Set up service for write-in adjudication

### LINKED ISSUE #1827: Set up service for write-in adjudication DB

### PR DESCRIPTION
**Pull request by taravancil**
## Overview
Closes #1827

This sets up a bare service for write-in adjudication. I've called it `adjudication`, but I'm also accepting suggestions for better names 🙂  

## Testing Plan 
Automated.

## Checklist
- [x] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [x] I have added JSDoc comments to any newly introduced exports

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Comment by eventualbuddha** [comment]
> I've called it `write-ins`, but I'm also accepting suggestions for better names

`adjudication`?

**Comment by taravancil** [comment]
> Or admin, if this is going to be the backend for VxAdmin?

I went with `adjudication` since we may have non-VxAdmin adjudication stations in the future. Does that work for you @eventualbuddha ?

**Comment by eventualbuddha** [comment]
> I went with `adjudication` since we may have non-VxAdmin adjudication stations in the future.

True, but most likely those stations would not have their own copy of this running but would communicate with the running on VxAdmin. Is that how you imagined it?

**Comment by taravancil** [comment]
That is, and you know what, `admin` is shorter and easier to type than `adjudication`, so let's do that.

File: services/admin/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "extends": ["plugin:vx/recommended"],
+  "rules": {
+    "vx/gts-jsdoc": "off"
```
Comment (eventualbuddha) [inline services/admin/.eslintrc.json:4]: Can we not do this in a new project?

File: services/admin/.gitignore [hunk 2]
```diff
@@ -0,0 +1,12 @@
+node_modules
+dev-workspace/
+hmpb-templates
+!hmpb-templates/.gitkeep
```
Comment (eventualbuddha) [inline services/admin/.gitignore:4]: Not relevant?

File: services/admin/.gitignore [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+*~
+*.log
+coverage/
+.DS_Store
+debug-dump*.zip
+/*.png
+/*.jpeg
+/*.jpg
```
Comment (eventualbuddha) [inline services/admin/.gitignore:12]: Not relevant?

File: services/admin/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+  "extends": "../../tsconfig.shared.json",
+  "include": ["src", "test"],
+  "exclude": [],
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "isolatedModules": false,
```
Comment (eventualbuddha) [inline services/admin/tsconfig.json:10]: We're probably going to want this to be `true` [for vite](https://vitejs.dev/guide/features.html#typescript-compiler-options), so let's just do that now.

File: services/admin/jest.config.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    '!src/index.ts',
+    '!src/types.ts',
+    '!test/**/*',
+  ],
+  coverageThreshold: {
+    global: {
+      statements: 90,
+      branches: 90,
+      functions: 90,
+      lines: 90,
```
Comment (eventualbuddha) [inline services/admin/jest.config.js:21]: Any reason not to leave these at the default 100?

File: services/admin/.eslintignore [hunk 6]
```diff
@@ -0,0 +1,5 @@
+build
+jest.config.js
+coverage
+
+public/vendor
```
Comment (eventualbuddha) [inline services/admin/.eslintignore:5]: Not relevant?

File: services/admin/.gitignore [hunk 7]
```diff
@@ -0,0 +1,12 @@
+node_modules
```
Comment (eventualbuddha) [inline services/admin/.gitignore:1]: This file should include `build`, right?

**Review by eventualbuddha — APPROVED** [review]
Some stuff to clean up and some questions, but overall good.

File: services/admin/.gitignore (hunk 7 above)
Comment (taravancil) [reply services/admin/.gitignore:1]: We have `build` in our top-level `.gitignore` so not strictly necessary here

**Pull request merged**